test/integration/api/workflow.spec.js

```diff
@@ -191,6 +191,15 @@ test('creates several single-step Stelace workflows', async (t) => {
           startDate: 'new Date().toISOString()',
           endDate: 'new Date(new Date().getTime() + (14 * 24 * 60 * 60 * 1000)).toISOString()',
           isCurrentTest: '_.get(changesRequested, "metadata.singleStepWorkflow")',
+
+          // localized date in Chinese format
+          localizedDate: `
+            new Intl.DateTimeFormat('zh-CH', {
+              day: 'numeric',
+              month : 'long',
+              year : 'numeric'
+            }).format(new Date('2019-02-05'))
```

Comment:
Thanks, we’ll need to mention Intl object in Workflow API reference.

Reply:
Unless we clearly state in our docs + API reference that our servers use UTC time.